### DISCUSSION TITLE
How to install on Linux and macOS

### DISCUSSION THREAD

**Discussion by perdrix52** [microsoft/vcpkg#42563] [General]
Please can you tell me the step by step instructions to install vcpkg on Linux and macOS?

Does it have to be installed to /home/_username_/vcpkg 

Or can it be installed for use by all users of the system to (e.g.) /opt/vcpkg

It seems rather limiting to require all users to install their own vcpkg instance and also a waste of disk resources if multiple users are logged in and are all working on the same project and each has their own copy of all the dependencies???  (yes I know they will have copies in the vcpkg_installed directory, but they will also have their own downloads/package/ etc dirs under vcpkg dir

**Comment by dg0yt** [comment]
> Please can you tell me the step by step instructions to install vcpkg on Linux and macOS?

Step by step install instructions: https://learn.microsoft.com/vcpkg/get_started/get-started?pivots=shell-bash#1---set-up-vcpkg



> Does it have to be installed to /home/_username_/vcpkg

No, you can choose another location, and you can have many clones.

> 

> Or can it be installed for use by all users of the system to (e.g.) /opt/vcpkg

This doesn't make much sense in the general case, and it is not how vcpkg is designed AFAICT.
(I acknowledge that your particular use case is "multiple users ... all working on the same project".)
